components/automate-ui/src/app/modules/infra-proxy/client-details/client-details.component.spec.ts

```diff
@@ -32,6 +32,7 @@ describe('ClientDetailsComponent', () => {
         MockComponent({ selector: 'chef-tr' }),
         MockComponent({ selector: 'chef-th' }),
         MockComponent({ selector: 'chef-td' }),
+        MockComponent({ selector: 'chef-snippet', inputs: ['code'] }),
```

Comment:
Some cleanup of missing mock-selectors.  I'm actually surprised these made it through bulidkite.